login/models.py

```diff
@@ -0,0 +1,3 @@
+# from django.db import models
```

Comment:
Is there any data exist in addition to django user model?

Reply:
No additional data models.